bridgenode/config.go

```diff
@@ -208,16 +216,13 @@ type Config struct {
 	forestType forestType
 
 	// quitAfter syncing to this block height
-	quitAt int
+	quitAfter int32
 
 	// how much cache to allow for cowforest
 	cowMaxCache int
 
-	// keep the ttldb in memory
-	memTTLdb bool
-
-	// only keep the ttldb in memory
-	allInMemTTLdb bool
+	// keep ttls in memory
```

Comment:
If we're taking out the option for caching in ram, this field should be gone in the config